### DISCUSSION TITLE
Welcome to qmk_firmware Discussions!

### DISCUSSION THREAD

**Discussion by JohnAZoidberg** [FrameworkComputer/qmk_firmware#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tzarc** [comment]
Hi there!

Normally QMK likes to have lines of communication open with larger manufacturers to ensure the upstreaming process goes as smoothly as possible. Some manufacturers leverage that better than others, so I figured jumping in early and establishing that communication in a general sense would hopefully result in a positive result for everybody.

QMK's development interactions are largely done on Discord -- if you could give me a ping directly we can set up a channel to ensure there's alignment with direction early... we realise there are "core" changes here, so making sure they're correctly implemented as far as QMK's future requirements go would be great.
